os/board/rtl8721csm/src/component/soc/realtek/amebad/fwlib/ram_hp/rtl8721dhp_app_start.c

```diff
@@ -1325,6 +1325,8 @@ void app_start(void)
 	app_section_init();
 	_memset((void *) __bss_start__, 0, (__bss_end__ - __bss_start__));
 
+	DBG_ERR_MSG_ON(MODULE_MISC); /* Enable DBG_8195A debug log */
```

Comment:
What is the purpose/meaning of this debug log?

Reply:
It is to show the default hardfault log for secure.
Because the handler is in ROM, so we cannot change the print, but we can enable by this variable.
But this this DBG_8195A is commonly use at other places, so there is no specific purpose/meaning.

Reply:
Could you write what you said in the commit description?
We can't what this change does with the title and the code change.

Reply:
yo mean add "Enable RTK debug message during app_start" into code comment?

Reply:
I think, `Enable debug for hard fault handler` or `Enable debug for secure fault` is better.
Maybe @sunghan-chang 's opinion is same.

Reply:
Modify it to "Enable debug log for hard fault handler"